Packages/Sites/Neos.NeosIo/Resources/Private/Templates/Page/Default.html

```diff
@@ -25,6 +25,7 @@
 		keyVisualImageSrcSet: keyVisualImageSrcSet
 	}" />
 
+	<nav>{parts.eventNav -> f:format.raw()}</nav>
```

Comment:
maybe wrap this in an <f:if condition="{parts.eventNav}"> ? otherwise we'll have an unnecessary <nav> element